core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -3448,7 +3445,7 @@ class KafkaApisTest {
     val expectedJoinGroupResponse = new JoinGroupResponseData()
       .setErrorCode(Errors.INCONSISTENT_GROUP_PROTOCOL.code)
       .setMemberId("member")
-      .setProtocolName(if (version >= 7) null else GroupCoordinator.NoProtocol)
+      .setProtocolName(if (version >= 7) null else kafka.coordinator.group.GroupCoordinator.NoProtocol)
```

Comment:
Did we want to move these constants too?

Reply:
Yes, we should but I wanted to do this separately because I don't know yet where to put them.